alert-policies/redmine/ErrorRate.yml

```diff
@@ -0,0 +1,36 @@
+name: Error Rate
+
+description: |+
+  This alert is triggered when the Error Rate is higher than 5 for 5 minutes.
```

Comment:
This alert is triggered when the average error rate is higher than 5 for 5 minutes.